src/Serilog.Sinks.File/Serilog.Sinks.File.csproj

```
@@ -4,7 +4,7 @@
     <Description>Write Serilog events to text files in plain or JSON format.</Description>
     <VersionPrefix>5.0.1</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netstandard2.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netstandard2.1;net5.0;net6.0;net7.0</TargetFrameworks>
```

See comment on PR with link re rationale for not (ever) adding `net7.0` here
```suggestion
    <TargetFrameworks>net45;netstandard1.3;netstandard2.0;netstandard2.1;net5.0;net6.0</TargetFrameworks>
```

---

NOTE this is on the assumption that we are aiming for a 5.0.1 or 5.1.0
If we were doing a 6.0 release of the sink:
- we support `netstandard2.0`, so no need to support `netstandard2.1`,`net5.0`,`net6.0`
- ... so I'd be proposing to simply remove `net5.0` and `netstandard2.1` here (f there is something that we support that benefits from either, then we add `net6.0` (and not `net8.0`))